src/app/components/AppsList.tsx

```
@@ -0,0 +1,97 @@
+    );
+    const appConfig = await config.text();
+    try {
+      const appConfigJson = JSON.parse(appConfig);
+
+      apps[app] = {
+        name: appConfigJson.name,
+        description: appConfigJson.short_desc,
+        logo: `https://raw.githubusercontent.com/runtipi/runtipi-appstore/master/apps/${app}/metadata/logo.jpg`,
+        link: appConfigJson.source,
+      };
+    } catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)